docker_images/generic/tests/test_api_tabular_regression.py

```diff
@@ -0,0 +1,96 @@
+import json
+import os
+from unittest import TestCase, skipIf
+
+from parameterized import parameterized_class
+from starlette.testclient import TestClient
+from tests.test_api import TESTABLE_MODELS
+
+
+@skipIf(
+    "tabular-regression" not in TESTABLE_MODELS,
+    "tabular-regresson not implemented",
+)
+@parameterized_class(
+    [
+        {"model_id": model_id}
+        for model_id in TESTABLE_MODELS["tabular-regression"]
+    ]
+)
+class TabularRegressionTestCase(TestCase):
+    def setUp(self):
+        self.old_model_id = os.getenv("MODEL_ID")
+        self.old_task = os.getenv("TASK")
+        os.environ["MODEL_ID"] = self.model_id
+        os.environ["TASK"] = "tabular-regression"
```

Comment:
are these env variables required inside some of the methods? Ideally you could just store them somewhere else or use original values if they're set. But I'm happy at least having some tests :)

Reply:
Yes, they are set in the docker image when spawned. @Narsil can give more details, but the answer is they are used indeed